playbooks/server-install.yml

```diff
@@ -135,3 +136,64 @@
         key: "{{ keys }}"
         state: present
         exclusive: true
+
+    - name: uninstall docker if it exists
+      shell: |
+        for pkg in docker.io docker-doc docker-compose podman-docker containerd runc; do apt-get remove $pkg; done
```

Comment:
If we make this change we can't use this same script to keep running against existing nodes. It'll technically be idompotent but it'll also break any running installations of docker. Currently I recall we can run the ansible script and not have downtime/outages. If i'm mistaken then this is probably fine. Also doing this like you did with systemd-resolved would be cleaner if we do keep it.